### PR TITLE
Update Kustomize doc for docker image

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
https://github.com/kubernetes-sigs/kustomize/issues/3168
https://github.com/kubernetes-sigs/kustomize/issues/3165

### CODE REVIEW

File: docs/404.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-<meta name="generator" content="Hugo 0.78.1" />
+<meta name="generator" content="Hugo 0.74.3" />
```
Comment (ricochet) [inline docs/404.html:6]: Weird this is older?

File: site/content/en/installation/kustomize/docker.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Docker Images"
+linkTitle: "Docker Images"
+weight: 3
+type: docs
+description: >
+  Install Kustomize by pulling docker images.
+---
+
+From kustomize v3.8.7, you can use docker to run kustomize. The images
```
Comment (ricochet) [inline site/content/en/installation/kustomize/docker.md:10]: I'd change to:
```md
Starting with Kustomize v3.8.7, docker images are available to run Kustomize.
The images are hosted in Kubernetes official GCR repositories.

See [GCR page] for available images.
```

The documented commands worked for me.

**Comment by Shell32-Natsu** [comment]
cc @monopole

**Pull request merged**